### PR TITLE
Makes cybernetic organs less bad

### PR DESCRIPTION
**Pull request by Moltijoe**
cybernetic organs are so bad that i've had people actively refuse them even if their current one is entirely non-functional

Synthetic organs don't passively heal, so if they get damaged, you basically just need to replace them
Doesn't change that, but instead gives them more health so they take longer to break
and makes them slightly more efficient than organic self-healing ones

regular cybernetic hearts now restart after a bit on emp so they aren't instant death
upgraded cybernetic hearts restart faster

might spend the time to give the upgraded ones additional effects so taking the upgraded cybernetic organ perk is actually worth it

:cl:  
tweak: All cybernetic and upgraded cybernetic organs have more health
tweak: cybernetic and upgraded cybernetic lungs/hearts are more efficient
tweak: Cybernetic hearts restart after a bit on emp
tweak: Upgraded cybernetic hearts restart faster on emp
tweak: IPC brains take 10 less damage from heavy emp, but give drugginess
tweak: syndicate leg implants can't break legs on emp
tweak: cybernetic ears knock down for slightly less time on emp
/:cl:

### CODE REVIEW

**Comment by MajManatee** [comment]
I would like to see the upgraded versions of implants be upgrades, rather than the sort of side grades to normal organs that they are now. The base version of cybernetic organs do, and should, kinda be mid IMO.

maybe experimental organs in a future pr.

File: code/modules/surgery/organs/augment_legs.dm [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	L.set_disabled(TRUE)	//disable the bodypart
 	addtimer(CALLBACK(src, .proc/reenableleg), 5 SECONDS, TIMER_UNIQUE|TIMER_OVERRIDE)
 
-	if(severity & EMP_HEAVY && prob(5))	//put probabilities into a calculator before you try fucking with this
+	if(severity & EMP_HEAVY && prob(5) && !syndicate_implant)	//put probabilities into a calculator before you try fucking with this
 		to_chat(owner, span_warning("The EMP causes your [src] to thrash your [L] around wildly breaking it!"))
```
Comment (ReddicusDragon) [inline code/modules/surgery/organs/augment_legs.dm]:
```suggestion
		to_chat(owner, span_warning("The EMP causes your [src] to thrash your [L] around wildly, breaking it!"))	
```

File: code/modules/surgery/organs/ears.dm [hunk 2]
```diff
@@ -158,11 +158,11 @@
 		if(1)
 			owner.Jitter(30)
 			owner.Dizzy(30)
-			owner.Knockdown(5 SECONDS)
+			owner.Knockdown(3 SECONDS)
 			deaf = 30
-			to_chat(owner, "<span class='warning'>Your robotic ears are ringing, uselessly.</span>")
+			to_chat(owner, "<span class='warning'>Your robotic ears are uselessly ringing.</span>")
```
Comment (ReddicusDragon) [inline code/modules/surgery/organs/ears.dm]:
```suggestion
			to_chat(owner, span_warning("Your robotic ears are uselessly ringing."))
```

File: code/modules/surgery/organs/ears.dm [hunk 3]
```diff
@@ -158,11 +158,11 @@
...
 		if(2)
 			owner.Jitter(15)
 			owner.Dizzy(15)
-			owner.Knockdown(10 SECONDS)
+			owner.Knockdown(6 SECONDS)
 			to_chat(owner, "<span class='warning'>Your robotic ears buzz.</span>")
```
Comment (ReddicusDragon) [inline code/modules/surgery/organs/ears.dm]:
```suggestion
			to_chat(owner, span_warning("Your robotic ears buzz.")) 
```

File: code/modules/mob/living/brain/brain_item.dm [hunk 4]
```diff
@@ -261,12 +261,13 @@
...
+
+	owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 50/severity)
+	owner.adjust_drugginess(40/severity)
 	switch(severity)
 		if(1)
-			owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 60)
 			to_chat(owner, "<span class='warning'>Alert: Posibrain heavily damaged.</span>")
 		if(2)
-			owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 25)
 			to_chat(owner, "<span class='warning'>Alert: Posibrain damaged.</span>")
```
Comment (ReddicusDragon) [inline code/modules/mob/living/brain/brain_item.dm]:
```suggestion
			to_chat(owner, span_warning("Alert: Posibrain damaged.")) 
```

File: code/modules/mob/living/brain/brain_item.dm [hunk 5]
```diff
@@ -261,12 +261,13 @@
 /obj/item/organ/brain/positron/emp_act(severity)
 	if(prob(25))
 		return
+
+	owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 50/severity)
+	owner.adjust_drugginess(40/severity)
 	switch(severity)
 		if(1)
-			owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 60)
 			to_chat(owner, "<span class='warning'>Alert: Posibrain heavily damaged.</span>")
```
Comment (ReddicusDragon) [inline code/modules/mob/living/brain/brain_item.dm]:
```suggestion
			to_chat(owner, span_warning("Alert: Posibrain heavily damaged."))
```

File: code/modules/surgery/organs/liver.dm [hunk 6]
```diff
@@ -102,7 +102,7 @@
 	icon_state = "liver-c-u"
 	desc = "An upgraded version of the cybernetic liver, designed to handle extreme levels of toxins. It can even heal minor amounts of toxin damage."
 	alcohol_tolerance = 0.0005
-	maxHealth = 1.5 * STANDARD_ORGAN_THRESHOLD //150% health of a normal liver
+	maxHealth = 3 * STANDARD_ORGAN_THRESHOLD //150% health of a normal liver
```
Comment (ReddicusDragon) [inline code/modules/surgery/organs/liver.dm]: is this still 150%?

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]

File: code/modules/surgery/organs/liver.dm (hunk 6 above)
Comment (Moltijoe) [inline code/modules/surgery/organs/liver.dm]:
```suggestion
	maxHealth = 3 * STANDARD_ORGAN_THRESHOLD //300% health of a normal liver
```

**Review by ReddicusDragon — APPROVED** [review]

**Comment by MajManatee** [comment]
@ReddicusDragon wtf speedmerge?!

**Pull request merged**